mix.exs

```diff
@@ -44,7 +44,7 @@ defmodule Membrane.ICE.Mixfile do
       {:membrane_funnel_plugin, "~> 0.7.0"},
       {:membrane_telemetry_metrics, "~> 0.1.0"},
       {:bunch, "~> 1.5"},
-      {:fake_turn, "~> 0.3.0"},
+      {:fake_turn, github: "membraneframework/fake_turn", branch: "feature/track-send-errors"},
```

Comment:
Since PR in `fake_turn` is merged, this dependency should be to the `master` branch